section3/reflection.md

```diff
@@ -1,13 +1,14 @@
 ## Section 3 Reflection
 
-1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
-
-2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
-
-3. In your own words, explain what an Array method is.
-
-4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
-
-5. If you had to teach a 5 year old about loops, how would you explain it?
-
-6. What questions do you still have about Arrays and loops?
+1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?  
+There were a few spots where I could really feel the need to work indepedently but it mostly felt nice/natural to put into practice things I already learned as I went.
+2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.  
+I use a to do list app called todoist and each to do list is probably stored in an array and those arrays are probably stored in a larger array. Maybe we're all just arrays. In this essay I will...
+3. In your own words, explain what an Array method is.  
+Array methods are set commands that allow us to alter or analyze the contents of an array in various ways.
+4. What is the purpose of a loop? When would we use a loop in conjunction with array vs. without?  
+A loop allows us to run a bit of code over and over while a condition remains true so we might use a loop in conjunction with an array when we want to run a bit of code on the items in an array or on different items in general. If we just want to run a bit of code a set number of times without passing different data in each time we might just run a for loop without an array.
+5. If you had to teach a 5 year old about loops, how would you explain it?  
+Imagine you are eating ice cream with a spoon. To eat it you're getting a spoon of the ice cream and putting it in your mouth over and over until the ice cream is gone right? That's a loop. You're doing the same thing over and over again until it doesn't make sense to do that same thing anymore in this example because you're out of ice cream. Let's go eat ice cream.
+6. What questions do you still have about Arrays and loops?  
+As I joked above it seems like a lot of web/mobile apps using nesting arrays. It seems like that would...break my brain? Unless I learned some way to kind of compartmentalize the array layers? So I guess...that.
```

Comment:
Yes 100%. Data can get very gnarly and complex. We refer to them as next collections - in Mod 1 you'll be introduced to object literals which are another type of collection that offer some other benefits than arrays can.

The style/formatting of those nested collections makes a big difference in making it readable -> our brains not imploding. And, it still can be hard!